File: NX_P/libs/Models/src/lib/Game.ts
import { Board } from "./Board";
import { Player } from "./Player";
import { Values } from "./Values";

export interface Game {
    id : string;
    endDate? : Date;
    playerSymbol : Map<string , Values>;
    player1: Player;
    player2:Player;
    winner? : Player;
    board : Board;
}

File: TodoApi/Models/Game.cs
using System.ComponentModel.DataAnnotations;

namespace TodoApi.Models;
public class Game
{

    public string Id { get; set; }
    public DateTime EndDate { get; set; }
    public Player Player1 { get; set; }
    public Player Player2 { get; set; }

    public Player Winner { get; set; }
    public Board Board { get; set; }
}